code/_onclick/hud/screen_alarms.dm

```diff
@@ -85,18 +82,17 @@
 		client.screen -= alert
 	qdel(alert)
 
-/obj/abstract/screen/alert
-	name = "Alert"
-	desc = "Something seems to have gone wrong with this alert, so report this bug please."
-	icon = 'icons/mob/screen_alarms.dmi'
-	icon_state = "default"
-	mouse_opacity = TRUE
-	var/timeout = null //If set to a number, this alert will clear itself after that many deciseconds
-	var/severity = null
-	var/override_alerts = FALSE //If it is overriding other alerts of the same type
-	var/alerttooltipstyle = null 
 
 // PRIVATE = only edit, use, or override these if you're editing the system as a whole
+#define SCREEN_ALARM_LIMIT 5 //Only change this if you're also touching the list bellow.
```

Comment:
Can't you just do `screen_alarms_lock.len` ?

Reply:
yes, we can